pom.xml

```diff
@@ -24,7 +24,7 @@
     <alpn.api.version>1.1.3.v20160715</alpn.api.version>
     <java.level>8</java.level>
     <!-- TODO: remove once SpotBugs issues are fixed. 57 so far -->
-    <spotbugs.failOnError>false</spotbugs.failOnError>
+    <spotbugs.skip>false</spotbugs.skip>
```

Comment:
Did you mean for this to be true instead?

Reply:
Yes, fixed